sql-parser/src/main/java/io/crate/sql/parser/Statement.g

```diff
@@ -735,6 +735,10 @@ bool
     | FALSE
     ;
 
+jobId
+    : STRING
```

Comment:
specify uuid format or catch illegal argument exception in statement constructor, and also add a test for this